lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -297,7 +297,7 @@ sub create_panel_from_json_template ($$$$$$) {
         # As it is a trailing comma inside a string, it's not a terrible issue, the string will be valid,
         # but it will have an unneeded trailing comma.
         # The group (\W) at the end is to avoid removing commas before an opening quote (e.g. for "field": true, "other_field": ..)
-        $panel_json =~ s/(?<!("|'|\]|\}|\d))\s*,\s*"(\W)/"$2/g;
+        $panel_json =~ s/(?<!("|'|\]|\}|\d))\s*,\s*"(\W)/"$2/sg;
```

Comment:
hh, this kind of stuff always looks simple at start, but you always find a bug later on :stuck_out_tongue_closed_eyes:

Reply:
yes... what's the meaning of "hh"?